cmd/stdiscosrv/apisrv.go

```diff
@@ -136,7 +136,13 @@ func (s *apiSrv) handler(w http.ResponseWriter, req *http.Request) {
 	}
 
 	if s.useHTTP {
-		remoteAddr.IP = net.ParseIP(req.Header.Get("X-Forwarded-For"))
+		// X-Forwarded-For can have multiple client IPs; split using the comma separator
+		forwardIPs := strings.Split(req.Header.Get("X-Forwarded-For"), ",")
```

Comment:
Could use `strings.Cut` here as we are only interested in the first element in the list

Reply:
Ah! I didn't know about `strings.Cut()`
Thanks for the tip; I've pushed a new commit to use this.